_includes/tutorials/survey-responses/ksql/code/tutorial-steps/dev/source.json

```diff
@@ -0,0 +1,29 @@
+{
+  "connector.class"    : "ServiceNowSource",
+  "name"               : "recipe-servicenow-survey-respondents",
+  "kafka.auth.mode"    : "KAFKA_API_KEY",
+  "kafka.api.key"      : "<my-kafka-api-key>",
+  "kafka.api.secret"   : "<my-kafka-api-secret>",
+  "kafka.topic"        : "survey-respondents",
+  "output.data.format" : "AVRO",
```

Comment:
I could be wrong - but since the source tables and stream specify the incoming value format as `JSON` I think the connector configs need to update `AVRO` -> `JSON`